src/dreyfus_index_updater.erl

```diff
@@ -89,3 +101,43 @@ load_docs(FDI, _, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
         false ->
             {ok, setelement(1, Acc, I+1)}
     end.
+
+purge_index(Db, IndexPid, Index) ->
+    {ok, IdxPurgeSeq} = clouseau_rpc:get_purge_seq(IndexPid),
+    Proc = get_os_process(Index#index.def_lang),
+    true = proc_prompt(Proc, [<<"add_fun">>, Index#index.def]),
+    FoldFun = fun(PurgeSeq, {Id, _Revs}, Acc) ->
+        case couch_db:get_full_doc_info(Db, Id) of
+            not_found ->
+                ok = clouseau_rpc:delete(IndexPid, Id),
+                Acc0 = Acc;
```

Comment:
just `Acc;` here. Rather than assign a variable in every path, it's more idiomatic to use the result of the case statement (i.e, `Acc0 = case couch_db...`).